.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: cpp
+dist: trusty
+sudo: true
+
+env:
+  matrix:
+    - CONFIG=DEFAULT
+    - CONFIG=ALL_ON
+    - CONFIG=ALL_OFF
+
+addons:
+  apt:
+    packages:
+      - libpcap-dev
+      - libunwind8-dev
+
+install:
+  - git clone --depth=1 --branch=mbedtls-2.5.1 https://github.com/ARMmbed/mbedtls.git $HOME/mbedtls && mkdir -p $HOME/mbedtls-build && pushd $HOME/mbedtls-build && cmake -DCMAKE_INSTALL_PREFIX=/usr/local -DUSE_SHARED_MBEDTLS_LIBRARY=ON -DUSE_STATIC_MBEDTLS_LIBRARY=OFF -DENABLE_PROGRAMS=OFF -DENABLE_TESTING=OFF $HOME/mbedtls && make && sudo make install && popd
+  - git clone --depth=1 --branch=rewrite https://github.com/wiringX/wiringX.git $HOME/wiringx && mkdir -p $HOME/wiringx-build && pushd $HOME/wiringx-build && cmake $HOME/wiringx && make && sudo make install && popd
+
+before_script:
+  - if [[ $CONFIG == "ALL_OFF" ]]; then sed "s/\(set(.*\) ON/\1 OFF/" -i CMakeConfig.txt; fi
+  - if [[ $CONFIG == "ALL_ON" ]]; then sed "s/\(set(.*\) OFF/\1 ON/" -i CMakeConfig.txt; fi
+
+script:
+  - mkdir build && pushd build && cmake .. && make && ./pilight-unittest && sudo make install
```

Comment:
`make install` only copies files to the filesystem, which should work as long as it is run with sudo. So IMO it isn't needed to test this, it just slows down the build a bit